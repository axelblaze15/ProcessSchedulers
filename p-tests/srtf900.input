1000000000000000000 2 1
10000000000000 2 2
1000000000 2 3
1000000 2 4
1000 2 5
10 5 1