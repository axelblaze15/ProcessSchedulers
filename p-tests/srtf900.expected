10 6 0 1
1000 18 11 16
1000000 13 7 11
1000000000 9 4 7
10000000000000 5 1 3
1000000000000000000 3 0 1
4 7